sdk/security/keyvault/azadmin/settings/client_test.go

```diff
@@ -48,8 +48,6 @@ func TestGetSetting_InvalidSettingName(t *testing.T) {
 	require.Nil(t, res.Value)
 	var httpErr *azcore.ResponseError
 	require.ErrorAs(t, err, &httpErr)
-	require.Equal(t, "UnknownError", httpErr.ErrorCode)
-	require.Equal(t, 500, httpErr.StatusCode)
```

Comment:
500 is a server error--which should be transient 😅--so it makes sense that MHSM now responds differently. We could probably just update this to require the new error on the assumption that MHSM's new behavior is the norm, however I think this is the right change because the specifics of the error aren't important in testing the client.

Reply:
Agree, specific error isn't important so we can just test if there was an error.